Launcher, Notification & Splash Screen icons update

ezhd opened the pull request:
There are some alternative resource files for launcher icon foreground you can play with.
Once decided, I can update notification and splash icons to the spec.

ezhd committed:
Launcher icon update

simonpoole commented:
The 45° version is definitely better for the launcher and I suspect the notification icon, for the splash screen probably not.

Could you generate 45° versions of the PNGs for pre-Android SDK 26?

ezhd commented:
As the background is full-bleed for adaptive icon, I will have to choose a shape to crop for a legacy icon PNGs. 
Which should it be? 
Circle, Squircle, Rounded square, No-bleed?

<img width="120" height="120" alt="legacy_circle" src="https://github.com/user-attachments/assets/6ffcafa2-0d0b-40be-ae45-dde1c5b4ba17" /><img width="120" height="120" alt="legacy_squircle" src="https://github.com/user-attachments/assets/f28b2fed-0159-441a-acec-fe228c4849b9" /><img width="120" height="120" alt="legacy_rounded" src="https://github.com/user-attachments/assets/cea482fe-cfc9-4f87-b9d4-18637ea04dd2" /><img width="120" height="120" alt="legacy_no_bleed" src="https://github.com/user-attachments/assets/3eeb57b6-9bca-43c8-b27c-a7d9baff99e7" />

ezhd commented:
Rebased to the current master. 
• styles.xml conflict has been resolved.
• There are 2 options for legacy icons. Squircle is the default.
• Added an animated splash screen following the official guidelines.
• Added monochrome launcher icon as well.

simonpoole commented:
I'm a bit confused now, the last drafts were with the compass offset to the left, see https://github.com/MarcusWolschon/osmeditor4android/pull/3018#issuecomment-3592499725, but the ones in the current PR are the symmetric ones (which in a perfect world would be OK, but as they are are too likely to be confused with a watch face).

ezhd commented:
Done!

simonpoole committed:
Update release notes

HolgerJeromin commented:
Play store still has old Logo...

goldfndr commented:
[Taginfo](https://taginfo.openstreetmap.org/projects/vespucci#overview) no longer shows a logo.

ezhd commented:
> Play store still has old Logo...

New icon for Google pay store located here: src/main/ic_playstore.png
Feel free to move it anywhere




> [Taginfo](https://taginfo.openstreetmap.org/projects/vespucci#overview) no longer shows a logo.

https://github.com/MarcusWolschon/osmeditor4android/blob/276855492a142863e2d04a9c5178ca3be5bafd0a/src/main/java/de/blau/android/presets/Preset.java#L1612
is referring deleted PNG. 
I suggest changing it to 
.../master/documentation/docs/vespucci_logo_small.png
